packages/turf-clusters/index.js

```diff
@@ -0,0 +1,271 @@
+/**
+ * Get Cluster
+ *
+ * @param {FeatureCollection|Feature[]} geojson GeoJSON Features
+ * @param {*} filter Filter used on GeoJSON properties to get Cluster
+ * @returns {FeatureCollection} Single Cluster filtered by GeoJSON Properties
+ * @example
+ * var geojson = turf.featureCollection([
+ *     turf.point([0, 0], {cluster: 0, foo: 'null'}),
+ *     turf.point([2, 4], {cluster: 1, foo: 'bar'}),
+ *     turf.point([3, 6], {cluster: 1}),
+ *     turf.point([5, 1], {0: 'foo'}),
+ *     turf.point([4, 2], {'bar': 'foo'})
+ * ]);
+ *
+ * // Create a cluster by Object
+ * var cluster1 = turf.getCluster(geojson, {cluster: 1});
+ *
+ * // Create a cluster by String
+ * var cluster2 = turf.getCluster(geojson, 'cluster');
+ *
+ * // Create a cluster by an Array of Strings
+ * var cluster3 = turf.getCluster(geojson, ['cluster', 'foo']);
+ */
+function getCluster(geojson, filter) {
+    // Validation
+    if (!geojson) throw new Error('geojson is required');
+    if (filter === undefined || filter === null) throw new Error('filter is required');
+    if (geojson.type === 'FeatureCollection') geojson = geojson.features;
+    if (!Array.isArray(geojson)) throw new Error('invalid geojson');
```

Comment:
`@turf/invariant` right? To test for a valid FeatureCollection.
> would support invalid geojson constructs, like an array of features.

Agreed.. 👍 That's my "cheeky" style, i'll drop it.